### ISSUE TITLE
Schema error due to `with_filetree` without FQCN

### ISSUE THREAD

**Issue by hemberger** [ansible/ansible-lint#2256]
##### Summary

I'm seeing a perplexing schema error when I use the `with_filetree` loop statement. If I change this to `with_community.general.filetree`, the error goes away. Even though FQCN's are recommended, it should still be possible to use `with_filetree`.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```
ansible [core 2.13.2]
  config file = None
  configured module search path = ['/home/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/user/.local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/user/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/user/.local/bin/ansible
  python version = 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0]
  jinja version = 3.1.2
  libyaml = True
```

```
ansible-lint 6.4.0 using ansible 2.13.2
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

Ubuntu 22.04

##### STEPS TO REPRODUCE

```yaml
---
- template:
    src: "{{ item.src }}"
    dest: "{{ item.path }}"
    mode: u=rw,g=r,o=r
  with_filetree: "{{ files/ }}"
  when: item.state == 'file'
```

##### Desired Behavior

No warning.

##### Actual Behavior

```
schema: 'block' is a required property (schema[tasks])
roles/my_role/tasks/main.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``playbook-schema``,
``tasks-schema``.

This rule is not skippable and stops further processing of the file.

Schema bugs should be reported towards (https://github.com/ansible/schemas) project instead of ansible-lint.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.
```
This doesn't seem like a schema bug, which is why I'm reporting it here.

Thanks for your time!

**Comment by ssbarnea** [comment]
That is by design an documented on readme of schemas at https://github.com/ansible/schemas and it will not change. You either adopt using FQCN or you will not be able to pass `schema` checks.

There is no way to disable parts of schema checks as that is not allowed by the schema validation library.

In fact there are discussions about making schemas even more strict, but we want to pace these over a longer period of time, so people have time to update their content.

PS. One of the major reason for making schemas so strict is for limiting their size and improve performance.

**Comment by hemberger** [comment]
Thanks for the information! I'm a fairly new user of ansible-lint, so it was not clear from the error message that `with_filetree` was the schema offender (in fact, I had to test my file line-by-line to figure this out). Is it possible for a more informative error message be supplied to the user?
